apps/api/src/purchase-orders/dto/create-purchase-order.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { IsDateString, IsString, IsArray, IsNumber, ValidateNested, ValidateIf, Min } from 'class-validator';
+import { Type } from 'class-transformer';
+
```

Comment:
Server-side validation is added through the libraries 'class-validator' and 'class-transformer' and via decorators making for super readable code.